modules/402-ingress-nginx/crds/doc-ru-ingress-nginx.yaml

```
@@ -1,741 +1,748 @@
+
+                  **По умолчанию** включены только `TLSv1.2`, `TLSv1.3` и самые новые алгоритмы шифрования.
+              disableHTTP2:
+                description: |
+                  Выключить ли HTTP/2.
+              config:
+                description: |
+                  Секция настроек Ingress-контроллера, в которую в формате `ключ: значение(строка)` можно записать [любые возможные опции](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/).
+
+                  > **Внимание.** Ошибка в указании опций может привести к отказу в работе Ingress-контроллера.
+
+                  > **Внимание.** Не рекомендуется использовать данную опцию, так как не гарантируется обратная совместимость или работоспособность Ingress-контроллера.
+              additionalHeaders:
+                description: |
+                  Дополнительные header'ы, которые будут добавлены к каждому запросу. Указываются в формате `ключ: значение(строка)`.
+              additionalLogFields:
+                description: |
+                  Дополнительные поля, которые будут добавлены в логи nginx. Указываются в формате `ключ: значение(строка)`.
+              resourcesRequests:
+                description: |
+                  Настройки максимальных значений CPU и memory, которые может запросить под при выборе узла (если VPA выключен, максимальные значения становятся значениями по умолчанию).
+                properties:
+                  mode:
+                    description: |
+                      Режим управления реквестами ресурсов.
+                  vpa:
+                    description: |
+                      Настройки VPA режима управления.
+                    properties:
+                      mode:
+                        description: |
+                          Режим работы VPA.
+                      cpu:
+                        description: |
+                          Настройки для CPU.
+                        properties:
+                          max:
+                            description: |
+                              Максимальное значение, которое может выставить VPA для реквеста к CPU.
+                          min:
+                            description: |
+                              Минимальное значение, которое может выставить VPA для реквеста к CPU.
+                      memory:
+                        description: |
+                          Значение для запроса memory.
+                        properties:
+                          max:
+                            description: |
+                              Максимальное значение, которое может выставить VPA для реквеста к memory.
+                          min:
+                            description: |
+                              Минимальное значение, которое может выставить VPA для реквеста к memory.
```

```suggestion
                              Максимальное значение, которое может выставить VPA для запроса к CPU.
                          min:
                            description: |
                              Минимальное значение, которое может выставить VPA для запроса к CPU.
                      memory:
                        description: |
                          Значение для запроса memory.
                        properties:
                          max:
                            description: |
                              Максимальное значение, которое может выставить VPA для запроса к memory.
                          min:
                            description: |
                              Минимальное значение, которое может выставить VPA для запроса к memory.
```

Аналогично. Уместна ли будет замена на «запроса»?